src/Tribe/Views/V2/Hooks.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Handles hooking all the actions and filters used by the module.
+ *
+ * To remove a filter:
+ * remove_filter( 'some_filter', [ tribe( Tribe\Events\Views\V2\Hooks::class ), 'some_filtering_method' ] );
+ * remove_filter( 'some_filter', [ tribe( 'views-v2.filters' ), 'some_filtering_method' ] );
+ *
+ * To remove an action:
+ * remove_action( 'some_action', [ tribe( Tribe\Events\Views\V2\Hooks::class ), 'some_method' ] );
+ * remove_action( 'some_action', [ tribe( 'views-v2.hooks' ), 'some_method' ] );
+ *
+ * @package Tribe\Events\Views\V2
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2;
+
```

Comment:
I think we should include Tribe__Events__Rewrite here. `use Tribe__Events__Rewrite as Rewrite;`